tiling.js

```diff
@@ -1665,7 +1836,7 @@ class Spaces extends Map {
     }
 
     animateToSpace(to, from, callback) {
-        inPreview = false;
+        inPreview = PreviewMode.NONE;
```

Comment:
We'll need to remember the state here, otherwise we won't be able to select the correct finishing animation.

Reply:
fixed.